src/containers/Profile/Profile.jsx

```diff
@@ -1,11 +1,29 @@
-import React from 'react'
+import React from "react";
+import "./Profile.scss";
+import Button from "../../components/Button/Button";
+import {auth} from "../../firebase";
+import {signOut} from "firebase/auth";
 
 // path: /profile
 
 const Profile = () => {
+  const logOut = () => {
+    signOut(auth)
+      .then(() => {
```

Comment:
You don't need these `.then()` they don't do anything of use.

Reply:
Removed!